swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPublic.qll

```diff
@@ -171,6 +171,20 @@ module Content {
 
     override string toString() { result = "Tuple element at index " + index.toString() }
   }
+
+  /** A field of an enum element. */
+  class EnumContent extends Content, TEnumContent {
+    private ParamDecl f;
+
+    EnumContent() { this = TEnumContent(f) }
+
+    /** Gets the declaration of the enum field. */
+    ParamDecl getField() { result = f }
```

Comment:
Should this be called `getParam` instead (since it's not actually getting a `FieldDecl`)?